express-error-handling/server/server.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars -- REMOVE ME */
+import 'dotenv/config';
+import pg from 'pg';
+import express from 'express';
+import ClientError from './client-error.js';
+import errorMiddleware from './error-middleware.js';
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+      select *
+        from "grades"
+    `;
+    const result = await db.query(sql);
+    const grades = result.rows;
+    res.json(grades);
+  } catch (err) {
+    next(err);
+    console.error(err);
+    res.status(500).json({ error: 'an unexpected error occurred' });
```

Comment:
The error middleware sends this. You don't need to do it here.

Make both of these changes throughout.